packages/wdio-browserstack-service/src/service.ts

```diff
@@ -27,6 +35,12 @@ export default class BrowserstackService implements Services.ServiceInstance {
         this._options = { ...DEFAULT_OPTIONS, ...options }
         // added to maintain backward compatibility with webdriverIO v5
         this._config || (this._config = this._options)
+        if (this._options.testObservability == false) this._observability = false
```

Comment:
have converted it to 
```
if (this._options.testObservability == false) {
    this._observability = false
}

Reason: We have default as true. We want to keep it true in case it is passed as `true` as well as when it is not passed at all (`undefined`). If it is explicitly passed at `false` then only we make it false. Hence the above condition

Reply:
How about adding `testObservability: true` to `DEFAULT_OPTIONS` (in constants.ts)?